parts/linux/cloud-init/artifacts/krustlet.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Krustlet
+ConditionPathExists=/usr/local/bin/krustlet-wasi
+
```

Comment:
can you add one here for the krustlet-fix-ca.sh script too?

Reply:
a ConditionPathExists I mean. we don't want the service to try to start without these files.

Reply:
like this? ConditionPathExists=/usr/local/bin/krustlet-fix-ca.sh